### DISCUSSION TITLE
Xbox series s/x

### DISCUSSION THREAD

**Discussion by Gingiee** [ndeadly/MissionControl#264] [Q&A]
Does the series X/S controllers work?

**Comment by ndeadly** [comment]
No, not yet. They require BLE support first. There's an issue tracking it [here](https://github.com/ndeadly/MissionControl/issues/135) where you can read further details and follow progress.

**Reply by Gingiee** [reply]
Is there a estimate when it's gonna come out???

**Reply by impeeza** [reply]
Please read. Seriously the  facts are clear.  You can open the code and makes the changes by yourself
